src/test/limitedmap_tests.cpp

```diff
@@ -27,7 +27,7 @@ BOOST_AUTO_TEST_CASE(limitedmap_test)
     // make sure that the size is updated
     BOOST_CHECK(map.size() == 1);
 
-    // make sure that the new items is in the map
+    // make sure that the new items are in the map
```

Comment:
It does indeed look like only a single item was added above. That would make this another correction to '..new item is in the map', correct?